media/js/base/site.js

```diff
@@ -64,10 +64,7 @@
             if (pf.indexOf("Mac") !== -1) {
                 return 'oldmac';
             }
-            if (pf === '' &&
-                    /Firefox/.test(ua) &&
-                    /Mobile|Tablet/.test(ua) &&
-                    !(/Android/.test(ua))) {
+            if (pf === '' && /Firefox/.test(ua)) {
```

Comment:
I don't think we should remove `pf === ''` as this was specific to Firefox OS. It would be good to get confirmation on what this value is, else we may be safer simply checking for `Viera`?